### PR TITLE
CDAP-12953 Fix NPE that happened when statement field had macro.

### PR DESCRIPTION
**Pull request by anwar6953**
https://issues.cask.co/browse/CDAP-12953

### CODE REVIEW

**Comment by sreevatsanraman** [comment]
Change looks good. Let's bump version and update market.

File: src/main/java/co/cask/hydrator/plugin/hive/action/HiveImport.java [hunk 1]
```diff
@@ -43,24 +42,31 @@ public HiveImport(HiveConfig config) {
...
+
+  private void validateHiveStatement(String statement) {
     //validate hive command. For import we only accept Load statements
-    if (!config.statement.toUpperCase().startsWith("LOAD")) {
+    if (!statement.toUpperCase().startsWith("LOAD")) {
       throw new IllegalArgumentException("To import data to a hive table, please use Hive LOAD query");
     }
 
-    if (config.statement.length() < 15) {
+    if (statement.length() < 15) {
```
Comment (gsps1) [inline src/main/java/co/cask/hydrator/plugin/hive/action/HiveImport.java]: the log messages could be a little more clear. like the error message for length, can say statement is longer than the allowed limit.

**Comment by gsps1** [comment]
LGTM

**Comment by anwar6953** [comment]
Bumped the project patch version and merging.

**Pull request merged**